neutron/templates/configmaps/openvswitch-ensure-configured.sh.yaml.j2

```diff
@@ -16,7 +16,7 @@ data:
 
     if [[ ! $(ovs-vsctl list-ports $bridge) =~ $(echo "\<$port\>") ]]; then
         changed=changed
-        #ovs-vsctl --no-wait add-port $bridge $port
+        ovs-vsctl --no-wait add-port $bridge $port
```

Comment:
I'm thinking on:
- create a l3-agent.sh.yaml.j2 
- call this script from l3-agent.yaml.j2 replacing the COMMAND
is that make sense ?